cartridges/int_adyen_SFRA/cartridge/forms/default/creditCard.xml

```diff
@@ -45,7 +45,7 @@
     <field formid="saveCard" label="label.save.card" binding="saveCard" type="boolean" mandatory="false"
            checked="false"/>
     <field formid="saveCardAdyen" label="label.save.card" binding="saveCardAdyen" type="boolean" mandatory="false"
-           checked="true"/>
+           checked="false"/>
```

Comment:
what does this do?

Reply:
Set the default value to true, but that need to be false